mvc/models.js

```diff
@@ -0,0 +1,8 @@
+const dataTest = require('../db/data/test-data/index')
+const dataDev = require('../db/data/development-data/index')
+const endpoints = require('../endpoints.json')
+const db = require('../db/connection')
+
+exports.getApiModel = () =>{
```

Comment:
We do't actually need a model here as we're not making a database request. We could just require the endpoints.json in the controller and send it from there 👍